server/task/task-distributed/src/main/scala/org/apache/james/task/eventsourcing/cassandra/CassandraTaskExecutionDetailsProjectionDAO.scala

```diff
@@ -26,12 +26,15 @@ import com.datastax.driver.core.{Row, Session}
 import javax.inject.Inject
 import org.apache.james.backends.cassandra.init.{CassandraTypesProvider, CassandraZonedDateTimeModule}
 import org.apache.james.backends.cassandra.utils.CassandraAsyncExecutor
+import org.apache.james.server.task.json.JsonTaskAdditionalInformationsSerializer
+import org.apache.james.task._
```

Comment:
do we have rules about wildcard import?

Reply:
I don't think that we have defined rules about wildcard import for scala code.
The alternative is :
`import org.apache.james.task.{Hostname, TaskExecutionDetails, TaskId, TaskManager, TaskType}`

By default intellij change the import to wildcard when there is at least 4 imports from the same package. 

What would you prefer ?

Reply:
I think that the default behavior of intellij is acceptable.

Reply:
Can you create a Scala guideline doc and put this decision in it?

Reply:
is there a checkstyle for Scala to auto document and ensure this? (it seems so http://www.scalastyle.org/)